src/numbers/Properties.java

```diff
@@ -0,0 +1,12 @@
+package numbers;
+
+public class Properties {
+
+    public static final String[] AVAILABLE_PROPERTIES
+            = {"even", "odd", "buzz", "duck", "jumping", "palindromic", "gapful", "spy", "square", "sunny", "happy", "sad"};
+    public static final String[][] MUTUALLY_EXCLUSIVE_PROPERTIES = {{"even", "odd"},
```

Comment:
Why array?

Reply:
Because there is no need to use features of more complicated collections.